src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -339,12 +355,14 @@ private static ExtHttpCall toExtHttpCall(CacheHttpCall cacheHttpCall) {
     }
 
     private Map<String, List<ExtBidderError>> toExtBidderErrors(List<BidderResponse> bidderResponses,
-                                                                BidRequest bidRequest, CacheServiceResult cacheResult) {
+                                                                BidRequest bidRequest, CacheServiceResult cacheResult,
+                                                                VideoStoredDataResult videoStoredDataResult) {
         final Map<String, List<ExtBidderError>> errors = new HashMap<>();
 
         errors.putAll(extractBidderErrors(bidderResponses));
         errors.putAll(extractDeprecatedBiddersErrors(bidRequest));
         errors.putAll(extractCacheErrors(cacheResult));
+        errors.putAll(extractStoredErrors(videoStoredDataResult));
```

Comment:
From what I can see below, you are putting `singletonMap(PREBID_EXT, extBidderErrors)` after `extractCacheErrors` puts `singletonMap(PREBID_EXT, Collections.singletonList(extBidderError))`. Therefore you might get a situation where there already is a `PREBID_EXT` key from cache errors, and `putAll` stored errors will try to add the same key which would simply override the existing value, resulting in cache errors being lost.

Please add an additional logic to add error for the pre-existing key.

Reply:
Thanks.